tests/subsys/usb/device_next/src/main.c

```diff
@@ -129,7 +129,7 @@ static void *usb_test_enable(void)
 	err = usbd_init(&test_usbd);
 	zassert_equal(err, 0, "Failed to initialize device support");
 
-	err = usbd_enable(&test_usbd);
+	err = usbd_enable(&test_usbd, true);
```

Comment:
Should we have a test where `wait` is `false` as well? Otherwise it seems completely untested

Reply:
There was a use in the sample, but I have dropped that commit.